be/src/exprs/json_functions.cpp

```diff
@@ -282,4 +282,67 @@ void JsonFunctions::get_parsed_paths(const std::vector<std::string>& path_exprs,
     }
 }
 
+Status JsonFunctions::extract_from_object(simdjson::ondemand::object& obj,
+                                          const std::vector<JsonPath>& jsonpath,
+                                          simdjson::ondemand::value* value) noexcept {
+#define HANDLE_SIMDJSON_ERROR(err, msg)                                                        \
+    do {                                                                                       \
+        const simdjson::error_code& _err = err;                                                \
+        const std::string& _msg = msg;                                                         \
+        if (UNLIKELY(_err)) {                                                                  \
+            if (_err == simdjson::NO_SUCH_FIELD || _err == simdjson::INDEX_OUT_OF_BOUNDS) {    \
+                return Status::NotFound(                                                       \
+                        fmt::format("err: {}, msg: {}", simdjson::error_message(_err), _msg)); \
```

Comment:
err: to simdjson err:

Reply:
what's the meaning of `err: to simdjson err:`?